samples/source/external-cluster/README.md

```diff
@@ -0,0 +1,153 @@
+# RabbitMQ Knative Eventing Source Example
+
+## Prerequisites
+
+- Same as listed [here](../../../docs/source.md#prerequisites)
+- A working RabbitMQ external instance exposed via an accessible IP/URL, an example on how to setup this can be found [here](./resources/rabbitmq.yml)
```

Comment:
Having an exchange and queue created is a prerequisite as well since the example is a predeclared one.

Reply:
Yes but I'm letting the user know that this could be done without predeclared resources

Reply:
You are absolutely right! fixed.